src/CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ install(TARGETS exiv2lib
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
-install(FILES ${LIBEXIV2_HDR}
+install(FILES ${LIBEXIV2_HDR} ${CMAKE_BINARY_DIR}/exv_conf.h
```

Comment:
exv_conf.h is already included in LIBEXIV2_HDR

Reply:
Right, good catch ;)